PatientWebApplication/UserMicroserviceApi/Model/User.cs

```diff
@@ -0,0 +1,43 @@
+/***********************************************************************
+ * Module:  User.cs
+ * Author:  Luna
+ * Purpose: Definition of the Class Model.Employee.User
+ ***********************************************************************/
+
+namespace UserMicroserviceApi.Model
+{
+    public class User : Entity
+    {
+        public string firstName { get; set; }
+        public string secondName { get; set; }
+        public string uniqueCitizensidentityNumber { get; set; }
+        public string dateOfBirth { get; set; }
+        public string phoneNumber { get; set; }
```

Comment:
Change properties names to start with upper letter and apply in rest of code (remove 'this' from constructors).